### ISSUE TITLE
NuGet

### ISSUE THREAD

**Issue by GrimaceOfDespair** [baks/EqualityTests#1]
I couldn't find any contact data, so I took the liberty of forking your project and creating a NuGet from it: https://www.nuget.org/packages/EqualityTests/

If you don't agree, or like to get the ownership of the NuGet, please let me know.

**Comment by baks** [comment]
@GrimaceOfDespair I wouldn't thought that someone want to use this code as a NuGet package.

Anyway, I'm more than happy that you find this code useful :)

**Comment by GrimaceOfDespair** [comment]
Well, it was a bit of a search, because I found your article wrongly linked, and was only able to find it by stripping the slash from the url (the link, which has probably changed over time, is both erroneous in google and in your readme). But: thanks for sharing all that!

I do plan to add an extra feature to detect members that are not taken into account. If I'm not mistaken, that's not covered now?

**Comment by baks** [comment]
You are right. Thank you for information. I've just fixed link to article in readme.

What do you mean to detect members which are not taken into account? Properties/Fields which are not assigned in constructor of type being tested?

**Comment by GrimaceOfDespair** [comment]
Correct. I know you targeted immutables, but it makes sense to support scenario's where you assume that all properties are defining.

Actually, it seems that there was [some activity](https://github.com/AutoFixture/AutoFixture/pull/353) on this in AutoFixture, but as far as I can tell, that has never reached the master. The PR was abandoned and then closed.

I did try out the proposed code over there, and though there were remarks on the PR, it does get the job done.

**Comment by baks** [comment]
Yes, I targeted `Value Objects` in definition within Domain-Driven Design. And I also saw that issue which you mention when I was working on that code. But I wanted to play with `AutoFixture`, hence this repository exists :smiley: 

If PR suits your needs then even better, because as we discussed code in this repository targets only value objects.
